contracts/staking/FixedStakingPool.sol

```diff
@@ -54,7 +54,7 @@ contract FixedStakingPool is Initializable, StakingPool, RoleAccessControl {
     function withdraw() external rewardsFinalized stakingPeriodComplete {
         UserInfo memory user = userInfo[_msgSender()];
         // checks
```

Comment:
nit; no ned for these inline comment.

_It's either clear that what follows are checks (as they are `require`) or the section can be moved into a `private` function following the clean code methodology._

_The extreme end of clean code, being everything in private methods and the top level function almost reads like a strange poem_